packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -210,7 +200,9 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
                     */
                     onBlur={(event) => {
                       const selectedLocation = allLocations.find((element) => element.display === event.target.value);
-                      setValue('location', { display: selectedLocation.display, uuid: selectedLocation.uuid });
+                      if (selectedLocation)
+                        setValue('location', { display: selectedLocation.display, uuid: selectedLocation.uuid });
+                      else setValue('location', { display: '', uuid: '' });
```

Comment:
This is to account for when you press the `x` option on a selected location option to clear the selection. Previously, this would cause an error since `selectedLocation` would be null when you do so